index.html

```diff
@@ -42,29 +42,40 @@ <h3 class="text-center">Create</h3>
         </div>
     </div>
     <div class="row">
-      <h3 class="text-center">Thanks to Our Sponsors</h3>
-      <p>   Code for San Jose enjoys the support of the following organizations: </p>
-          <ul>
-            <li>
-              organizational support from <a href="https://codeforamerica.org">Code for America</a>
-            </li>
-            <li>
-              financial support from <a href="https://www.microsoft.com">Microsoft</a>
-            </li>
-            <li>financial support from the <a href="http://www.awesomefoundation.org/">Awesome Foundation</a>
-            </li>
-            <li> financial support from the <a href="http://www.sanjoseca.gov/mayor">Office of Mayor Sam Liccardo</a>
-            </li>
-            <li> venue support from The Tech Museum’s <a href="https://www.thetech.org/education/forteachers/bowersinstitute">Bowers Institute</a>
+      <div class='col-sm-9'>
+        <h3 class="text-center">Thanks to Our Sponsors</h3>
+        <p>   Code for San Jose enjoys the support of the following organizations: </p>
+        <ul>
+          <li>
+            organizational support from <a href="https://codeforamerica.org">Code for America</a>
+          </li>
+          <li>
+            financial support from <a href="https://www.microsoft.com">Microsoft</a>
+          </li>
+          <li>financial support from the <a href="http://www.awesomefoundation.org/">Awesome Foundation</a>
+          </li>
+          <li> financial support from the <a href="http://www.sanjoseca.gov/mayor">Office of Mayor Sam Liccardo</a>
+          </li>
+          <li> venue support from The Tech Museum’s <a href="https://www.thetech.org/education/forteachers/bowersinstitute">Bowers Institute</a>
             <li> venue support from <a href="http://www.myactionspot.com/" target="_blank">ActionSpot</a> Start Up Studio
             </li>
           </ul>
-        <p>
-        Organizations interested in renting out the amazing co-working community and startup incubator space, ActionSpot should contact <a href="mailto:olga@myactionspot.com">Olga Buchonina</a>
-        </p>      
-        <p>
-        Organizations interested in renting out the beautiful Bowers Institute space should contact <a href="mailto:abault@thetech.org">Alison Wilhelm</a>.
-        </p>
+          <p>
+            Organizations interested in renting out the amazing co-working community and startup incubator space, ActionSpot should contact <a href="mailto:olga@myactionspot.com">Olga Buchonina</a>
+          </p>
+          <p>
+            Organizations interested in renting out the beautiful Bowers Institute space should contact <a href="mailto:abault@thetech.org">Alison Wilhelm</a>.
+          </p>
+      </div>
+      <div class="col-sm-3">
+        <a class="twitter-timeline"
+        href="https://twitter.com/codeforsanjose?ref_src=twsrc%5Etfw"
+        data-width="300"
+        data-height="400">
+        Tweets by codeforsanjose</a>
```

Comment:
How does this look on a mobile device? The site right now is responsive, so would be good to do a quick check on a phone to make sure it doesn't break the layout there.

Reply:
This is what I have when testing it in Responsive Mode - Apple iPhone SE

https://screenshots.firefox.com/kSsrtEMf0XirStWG/127.0.0.1